xrpl/helpers/converters.go

```diff
@@ -0,0 +1,62 @@
+package helpers
+
+import (
+	"encoding/hex"
+	"strings"
+)
+
+// StringToHex converts a string to a hexadecimal representation.
+// If isNonStandardCurrency is true, the hexadecimal representation is padded
+// with trailing zeros up to a length of 40 characters. This is to support the non-standard currency codes for the XRPL.
+// See https://xrpl.org/docs/references/protocol/data-types/currency-formats#nonstandard-currency-codes
+func StringToHex(input string, isNonStandardCurrency bool) string {
```

Comment:
Function should be renamed to `CurrencyStringToHex`

Reply:
The thing is that it might be used for something else than currencies, thus the name `StringToHex`.
In xrpl.js we have similar function (starting with `convert`):
<img width="260" alt="image" src="https://github.com/user-attachments/assets/5d1451fd-3d93-4841-874c-3d92f4575eeb">

I can rename it as you suggested and make the functions more dedicated to the currencies only but I wanted to have your opinion first.

Reply:
@florent-uzio If the function aims to convert a currency code from hex to string then it should be renamed as @GuillemGarciaDev says. However, if it is only a converter from hex to string which has nothing to do with currencies, like the xrpl.js is (https://github.dev/XRPLF/xrpl.js/), then I would suggest leaving the name as it is right now.

As a side note, I would avoid any usage of hex decoded strings in the code and use go native types instead (byte and string)